packages/generate-password/src/test/index.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * @jest-environment jsdom
+ */
+// eslint-disable-next-line import/no-nodejs-modules
+import { webcrypto } from 'node:crypto';
+import { generatePassword } from '../';
+
+describe( 'generatePassword', () => {
+	const mockRandomValues = jest.fn( ( array ) => webcrypto.getRandomValues( array ) );
+
+	Object.defineProperty( window, 'crypto', {
+		value: {
+			getRandomValues: mockRandomValues,
+		},
+	} );
```

Comment:
You did it again, jsdom 😅 Bummer that we have to use that.

I wonder if it makes sense to use `node:crypto` as a fallback, and that way make the package SSR-compatible? Maybe that'd be over-engineering, so I'm just thinking out loud.